packages/fast-usdc/src/cli/cli.js

```diff
@@ -0,0 +1,159 @@
+import { Command } from 'commander';
+import { existsSync, mkdirSync, readFileSync } from 'fs';
+import { fileURLToPath } from 'url';
+import { dirname, resolve } from 'path';
+import { homedir } from 'os';
+import {
+  readFile as readAsync,
+  writeFile as writeAsync,
+} from 'node:fs/promises';
+import configLib from './config.js';
+import transferLib from './transfer.js';
+import { makeFile } from '../util/file.js';
+
+const packageJson = JSON.parse(
+  readFileSync(
+    resolve(dirname(fileURLToPath(import.meta.url)), '../../package.json'),
+    'utf8',
+  ),
+);
+
+const defaultHome = homedir();
+
+export const initProgram = (
+  configHelpers = configLib,
+  transferHelpers = transferLib,
```

Comment:
is it useful to inject these 2 any more? Don't we just inject `fetch` and `makeSigner` and such?

**note**: ok... I get it... we mock `transferHelpers` to check that `initProgram` calls `transferHelpers` as expected and then unit test `transferHelpers` such as `.transfer`. fair enough.